backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/navigationbar/BpkTopNavBar.kt

```diff
@@ -35,6 +35,7 @@ sealed interface NavIcon {
 enum class NavBarStyle {
     Default,
     OnImage,
+    OnDark
```

Comment:
rather than onDark can you name it SurfaceContrast to match the figma name?
also nit: can you include the trailing comma please?

Reply:
Done!

Reply:
@marianeum - Curious to know "why it needs the trailing comma?" :)

Reply:
It's a minor thing: Adding a new enum value will only have one line change, rather than two making reviewing ever so slightly easier

Reply:
Got it, thanks.